corehq/apps/app_manager/dbaccessors.py

```diff
@@ -176,14 +177,20 @@ def get_app_cached(domain, app_id):
     """Cached version of ``get_app`` for use in phone
     api calls where most requests will be for app builds
     which are read-only.
-
     This only caches app builds."""
+    timeout = 24 * 3600
+    # cache to save in localmemory and then in default cache (redis)
+    #   Invalidation is not necessary as the builds don't get updated
+    cache = tiered_django_cache([
+        ('locmem', timeout, None),
+        ('default', timeout, lambda *x: "get_app_cached")]
```

Comment:
what's the purpose of the session function here?

Reply:
I am using it more as of a prefix, to help identify the source in redis.

Reply:
The cache key already has a prefix.